pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sml_small"
-version = "1.1.0rc10"
+version = "1.1.0"
```

Comment:
No rc?

Reply:
This relates to the official release so its no longer a release candidate